src/Reveal.Sdk.Dom.Tests/Visualizations/OHLCVisualizationFixture.cs

```diff
@@ -0,0 +1,361 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Reveal.Sdk.Dom.Core.Serialization;
+using Reveal.Sdk.Dom.Data;
+using Reveal.Sdk.Dom.Filters;
+using Reveal.Sdk.Dom.Visualizations;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Visualizations;
+
+public class OHLCVisualizationFixture
+{
+    [Fact]
+    public void Constructor_InitializesDefaultValues_WhenInstanceIsCreated()
+    {
+        // Act
+        var ohlcVisualization = new OHLCVisualization();
+
+        // Assert
+        Assert.NotNull(ohlcVisualization);
+        Assert.Equal(ChartType.OHLC, ohlcVisualization.ChartType);
+        Assert.Null(ohlcVisualization.Title);
+        Assert.Null(ohlcVisualization.DataDefinition);
+        Assert.NotNull(ohlcVisualization.Closes);
+        Assert.Empty(ohlcVisualization.Closes);
+        Assert.NotNull(ohlcVisualization.Highs);
+        Assert.Empty(ohlcVisualization.Highs);
+        Assert.NotNull(ohlcVisualization.Lows);
+        Assert.Empty(ohlcVisualization.Lows);
+        Assert.NotNull(ohlcVisualization.Opens);
+        Assert.Empty(ohlcVisualization.Opens);
+        Assert.Equal(0, ohlcVisualization.ColumnSpan);
+        Assert.Equal(0, ohlcVisualization.RowSpan);
+        Assert.Null(ohlcVisualization.Description);
+        Assert.NotNull(ohlcVisualization.Labels);
+        Assert.Empty(ohlcVisualization.Labels);
+        Assert.Null(ohlcVisualization.Linker);
+        Assert.True(ohlcVisualization.IsTitleVisible);
+    }
+
+    [Theory]
+    [InlineData(true)]
+    [InlineData(false)]
+    public void Constructor_InitializesOHLCVisualizationWithDataSourceItem_WhenDataSourceItemIsProvided(
+        bool hasTabularData)
+    {
+        // Arrange
+        var dataSourceItem = new DataSourceItem { HasTabularData = hasTabularData };
+
+        // Act
+        var ohlcVisualization = new OHLCVisualization(dataSourceItem);
+
+        // Assert
+        Assert.NotNull(ohlcVisualization);
```

Comment:
Remove this, b/c the codes below already ensure that ohlcVisualization is not null